.chloggen/mx-psi_literal-host.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component (e.g. pkg/quantile)
+component: pkg/otlp/attributes
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Handle literal 'host' attribute to avoid double tagging.
+
+# The PR related to this change
+issues: []
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: This avoids double tagging when using `metrics::resource_attributes_as_tags` metrics, or in all cases for traces/logs.
```

Comment:
What is double tagging? Is someone reading this expected to know? The same question for all inline comments in this PR. Why is it assumed that a newcomer/reader of this PR will know what that is? What is "double tagging"? Sounds like something impossible 😓

Reply:
Never mind, it is better explained downstairs. Resolving :)